index.html

```diff
@@ -39,8 +39,205 @@ <h1 class="hero-title" >I’m Mariane <br>
+                <!-- work 3  -->
+            <article class="work ">
+                <img class="work-main-img" src="./assets/Snapshoot Portfolio3.svg" alt="">
+                <aside class="work-info">
+                    <h2 class="work-title" >Tonic</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+                <!-- work 4  -->
+            <article class="work ">
+                <img class="work-main-img" src="./assets/Snapshoot Portfolio4.svg" alt="">
+                <aside class="work-info">
+                    <h2 class="work-title" >Multi-Post Stories</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about ">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
```

Comment:
- [ ] Kindly style your header tag to look like the figma design. Currently your font size is smaller than what is in the design guideline. So please refer and correct it
Your design
![image](https://user-images.githubusercontent.com/102699958/219572737-bb3c934a-241d-4b26-95f0-9693b56bb7cb.png)

The Figma design
![image](https://user-images.githubusercontent.com/102699958/219572986-96ba8937-a25a-42d5-b221-e82a24378783.png)